### DISCUSSION TITLE
Unable to clone this repository

### DISCUSSION THREAD

**Discussion by aishikaranjan** [Akshat111111/Hedging-of-Financial-Derivatives#604] [Q&A] [answered]
After cloning I get the following error as attached:

![image](https://github.com/user-attachments/assets/d648273d-4e1e-475d-8019-593c49df8ebe)

I've tried using the '--no-checkout'  option but it doesn't work either. 
Kindly help in resolving.

**Comment by Akshat111111** [comment, marked as answer]
You can try using github codespace
